pkg/scannerv4/mappers/mappers.go

```diff
@@ -104,7 +113,11 @@ func ToProtoV4VulnerabilityReport(ctx context.Context, r *claircore.Vulnerabilit
 	if err != nil {
 		return nil, fmt.Errorf("internal error: parsing nvd vulns: %w", err)
 	}
-	vulnerabilities, err := toProtoV4VulnerabilitiesMap(ctx, r.Vulnerabilities, nvdVulns)
+	cveEPSS, err := cveEPSS(r.Enrichments, ctx)
+	if err != nil {
+		return nil, fmt.Errorf("internal error: parsing package-level EPSS details: %w", err)
```

Comment:
Why the EPSS details are considered package-level? I thought they were reported per CVE.

Reply:
I've seen a few references in this PR to EPSS being related to packages, which I'm confused about. EPSS is package agnostic, only relates to CVEs